app/src/main/res/layout/activity_all_saved_outfits.xml

```diff
@@ -7,86 +7,101 @@
     tools:context=".closetPage.AllSavedOutfits"
     android:background="#FFE0E7">
 
-    <RelativeLayout
-        android:id="@+id/relativeLayout"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+
+
+    <!--
```

Comment:
If this code is not used I think deleting the comment would make the pages look nicer.